ci/azure-steps-windows.yml

```diff
@@ -0,0 +1,26 @@
+steps:
+- powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+  displayName: Add conda to PATH
+
+- script: |
+    conda config --set always_yes True
+    conda update -n base conda
+    conda init
+    conda env update -n uarray -f .conda/environment$(python_version).yml
```

Comment:
FYI Windows+3.7 can fail because of this bug:

https://github.com/conda/conda/issues/8187

But if your `pip`-installed deps don't require `numpy` you might be okay

Reply:
They don't.

Reply:
I wouldn't depend on pip at all when using `conda` - you can avoid a whole class of bugs and bad interaction by simply submitting PRs to [`staged-recipes`](https://github.com/conda-forge/staged-recipes) to get all your deps available with `conda`

Reply:
@dhirschfeld The 3.5 deps don't include pip and it still fails... Because it doesn't recognize type hints.